extensions/spring-cloud-config-client/runtime/src/main/java/io/quarkus/spring/cloud/config/client/runtime/DefaultSpringCloudConfigClientGateway.java

```diff
@@ -191,6 +191,7 @@ private URI finalURI(String applicationName, String profile) throws URISyntaxExc
         List<String> finalPathSegments = new ArrayList<>(result.getPathSegments());
         finalPathSegments.add(applicationName);
         finalPathSegments.add(profile);
+        springCloudConfigClientConfig.label.ifPresent(label -> finalPathSegments.add(label));
```

Comment:
We generally try to avoid having lambdas in code that executes at runtime.
Can you please replace it with an `if` statement?

Reply:
Okay, replaced the lambda with an `if` statement.